extensions/sql-bindings/src/common/telemetry.ts

```diff
@@ -2,10 +2,10 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import * as vscode from 'vscode';
 import AdsTelemetryReporter from '@microsoft/ads-extension-telemetry';
+import { getPackageInfo } from './utils';
 
-const packageInfo = vscode.extensions.getExtension('Microsoft.sql-bindings')?.packageJSON;
+const packageInfo = getPackageInfo()!;
```

Comment:
Another option we could utilize if we are just differentiating between ads and vscode runtime would be setting up a enum for extension (similar to [this](https://github.com/microsoft/azuredatastudio/blob/main/extensions/data-workspace/src/dataworkspace.d.ts#L9) and based on a getAzdataAPI() call we can set `vscode.extensions.getExtension` with the correct extension name.

Reply:
You should be able to get this from the extension context 

`extensionContext.extension.packageJSON`

Reply:
But given that it would require rewriting this stuff to be initialized on startup I guess this is ok. Pretty terrible pattern but not the worst to leave like this for now

Reply:
Yeah, I tried to work on this method after looking more into the extension context. But saw it was gonna require quite a bit of refactoring as well. 

So do you think what I have now is good or should I change to the option I described above?